.eslintrc

```diff
@@ -4,5 +4,8 @@
     "comma-dangle": ["error", "always-multiline"],
     "indent": ["error", 2, { "ignoredNodes": ["ConditionalExpression"] }],
     "semi": ["error", "always"]
+  },
+  "parserOptions": {
+    "ecmaVersion": 2017
```

Comment:
I added this because I was getting linting errors when trying to write tests using `async/await`

Reply:
This is because the Rascal codebase style is rather old. It's long overdue for modernisation though.